### DISCUSSION TITLE
Localized Pop-up for Chat Rules/Game Rules

### DISCUSSION THREAD

**Discussion by msmcculley** [ppy/osu#14830] [Ideas]
Hey all, 

I was wondering if there was a plan or an idea to implement a pop-up/ToS/something that would introduce or define the chat rules for the first time that a player were to open the chat, either the first time or on every first time that playing session, in their respective language that can be detected by the settings or however osu! usually does. If not then the language is easily changeable in the settings. 

Some of the main issues that I see in #osu (specifically):
>**New players speaking in their native tongue,** either on accident, or then on purpose after the chat tries to direct them to the right channel. (They seem to not understand that #osu is english only) 
>**Chat topics** can be a disaster because not everyone knows that osu! is for all ages and some topics are just not suitable for the younger crowd. 
>**General chat rules like:** Spamming, flooding, creating a ladder game, don't be sexist, racist, homophobic, transphobic, (In general just being nice and a respectful human being).
>Other smaller issues about streaming, correct channels, correct language for the channel you are currently in, etc. 
**>Another big issue that is more of an annoyance than anything is the constant flood of multiplayer lobbies being broadcasted into #osu. It has become such an issue that the only way that people would ever listen to this is if some how banchobot/the rule system of #osu would immediately remove the message and put it into #lobby or flat out remove the text and bancho sends the user the !faq multiplayer command or the !faq lobby faq. 

I believe there has been too much leniency with what can go into #osu and what can't under the assumption people will read the rules or what not. ** 

**Text that was made after the initial post
Anyways, thanks for the read and hopefully I made this clear enough for the right people to see.

**Comment by TicClick** [comment]
that's a good idea; while it's still possible to click through the popup without reading, its existence would somewhat lower the amount of bad behaviour usually seen in the chat (at least from the new players)

**Comment by Nukrid** [comment]
Even considering most people would close it without reading, if even 10% of the new players read it properly (which to be fair, are the biggest "troublemakers" of this day and age) it would decrease the amount of spam and misbehavior we usually see.

**Comment by msmcculley** [comment]
Thought about this a little more and figured I’d bump it to see if the team had anything to add as this was made quite some time ago

**Reply by peppy** [reply]
I'm not sure what answer you're looking for, but nothing has really changed. We have other priorities.
